dist-git/conf/copr-dist-git.sysusers.conf

```diff
@@ -1,7 +1,7 @@
 #Type Name     ID             GECOS                 Home directory Shell
 g     packager -
 g     copr-dist-git -
-u     copr-dist-git -         'copr-dist-git user'  -              -
+u     copr-dist-git -         'copr-dist-git user'  /home/copr-dist-git  /bin/bash
```

Comment:
Is it correct to create these tmpfiles on non-tmpfs filesystem?  That's the first time I see this being done :shrug: people normally executed `mkhomedir_helper`

Reply:
Argh, sorry.  I thought I'm commenting on the tmpfiles.d snippet.
Yes, we probably need to specify homedir explicitly, just not sure if we want /home sub-dir.

Reply:
I will test `/var/lib/copr-dist-git` that you suggested on Slack.

Reply:
Worked fine, I updated the PR.